Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -265,14 +242,10 @@ AppDelegateViewModelOutputs {
       .filter { AppEnvironment.current.currentUser != nil }
 
     if #available(iOS 10.0, *) {
-      self.getNotificationAuthorizationStatus =
-        Signal.merge(
-          applicationLaunchedWithUser,
-          self.notificationAuthorizationCompletedProperty.signal)
+      self.getNotificationAuthorizationStatus = applicationLaunchedWithUser
```

Comment:
I believe we want to get the notification settings only on app launch (etc), not after authorization has been requested. I didn't see this pattern documented anywhere, but open for debate.